modules/nf-core/ascat/tests/main.nf.test

```diff
@@ -0,0 +1,43 @@
+nextflow_process {
+
+    name "Test Process ASCAT"
+    script "../main.nf"
+    process "ASCAT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "ascat"
+
+    test("ascat") {
+
+        config "./simple-nextflow.config"
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id: 'test', single_end:false ],
+                    [
+                        file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true),
+                        file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam_bai'], checkIfExists: true),
+                        file(params.test_data['homo_sapiens']['illumina']['test2_paired_end_sorted_bam'], checkIfExists: true),
+                        file(params.test_data['homo_sapiens']['illumina']['test2_paired_end_sorted_bam_bai'], checkIfExists: true)
+                    ]
+                ]
+                input[1] = []
+                input[2] = []
+                input[3] = []
+                input[4] = []
+                input[5] = []
+                input[6] = []
+                """
+            }
+        }
+
+        then {
+            assertAll (
+                { assert process.success }
```

Comment:
can you add a check for the version?

Reply:
Added; but unfortunately the test doesn't work, I get the following error: `Path value cannot be null`

Reply:
when you ru the module manually do you get a versions file?

Reply:
I get the error when it runs `{ assert process.success }` so it doesn't run the test at all - could be something wrong with the way I declare the inputs? 

```
                input[1] = []
                input[2] = []
                input[3] = []
                input[4] = []
                input[5] = []
                input[6] = []
```

Reply:
the pytest runs with the following:

`ASCAT_SIMPLE ( input , [], [], [], [], [], [])`

Reply:
`Path value cannot be null` issue solved by @sateeshperi - but test still not working